packages/peregrine/lib/talons/OrderHistoryPage/orderRow.gql.js

```diff
@@ -1,5 +1,18 @@
 import { gql } from '@apollo/client';
 
+// We disable linting for local fields because there is no way to add them to
+// the fetched schema.
+// https://github.com/apollographql/eslint-plugin-graphql/issues/99
+/* eslint-disable graphql/template-strings */
+export const GET_CONFIGURABLE_THUMBNAIL_SOURCE = gql`
+    query getConfigurableThumbnailSource {
+        storeConfig {
+            id
+            configurable_thumbnail_source @client
+        }
+    }
+`;
+
```

Comment:
When you disable an eslint rule make sure to re-enable it immediately after the block you are ignoring, otherwise it will ignore the rest of the file.
```suggestion
/* eslint-enable graphql/template-strings */
```